guide-sharding-and-replication.md

```diff
@@ -127,6 +127,10 @@ kind: ClickHouseInstallation
 metadata:
   name: jaeger
 spec:
+  defaults:
```

Comment:
change to this CR is not related to this PR but it fixes using of persistent volumes.